docs/options.md

```diff
@@ -0,0 +1,30 @@
+# Changing PostgreSQL Options
+
+You may require a configuration change for your application. PostgreSQL
+allows the option to configure the database with a configuration file, as many other database
+management systems do. You can pass options to PostgreSQL instances using the
+specific Custom Resource option via the `deploy/cr.yaml` configuration file.
+
+Often there's no need to add custom options, as the Operator takes care of
+providing PostgreSQL with reasonable defaults.
+
+The `patroni` section in the Custom Resource, present in `deploy/cr.yaml` file,
+contains configuration options to customize the PostgreSQL high-availability
+implementation based on [Patroni](https://patroni.readthedocs.io/).
+
+```yaml
+...
+patroni:
+  dynamicConfiguration:
+    postgresql:
+      parameters:
+        max_parallel_workers: 2
+        max_worker_processes: 2
+        shared_buffers: 1GB
+        work_mem: 2MB
+```
+
+Please note that configuration changes are automatically applied to the running instances
+without validation, so having an invalid config can make the cluster unavailable. 
+Also, take into account that some PosgreSQL options can't be applied dynamically but
+require restarting to be applied.
```

Comment:
We need way more meat here:

1. What would happen if I apply wrong config? 
2. How can I understand that I applied the wrong config?
3. What should I do if I applied the wrong config? How do I recover?
4. If I change only dynamic options - will we restart Pods / PostgreSQL process? If we restart it always - why do we mention it?

Reply:
Answering my own questions:
1. nothing, it will not be applied, old config stays
2. in the logs you will see that operator read the config, but changes were not applied
3. nothing
4. that I'm not sure and need pg experise. But judging by our option name we can apply only dynamic. pls verify

Reply:
@spron-in are you sure wrong config is not applied? do you think crunchy controller performs a validation and silently rejects wrong config? following comments on their code suggest me otherwise:

https://github.com/percona/percona-postgresql-operator/blob/main/pkg/apis/postgres-operator.crunchydata.com/v1beta1/patroni_types.go#L19-L27

https://github.com/percona/percona-postgresql-operator/blob/main/pkg/apis/postgres-operator.crunchydata.com/v1beta1/pgadmin_types.go#L54-L58

https://github.com/percona/percona-postgresql-operator/blob/main/pkg/apis/postgres-operator.crunchydata.com/v1beta1/pgbouncer_types.go#L69-L74